test/es7/asyncawait-undodefer.js

```diff
@@ -0,0 +1,36 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+async function asyncFunc1(a) {
+    return a * a;
+}
+
+asyncFunc1(10).then(
+    result => print(result),

```

Comment:
Due to the async nature I have to create a baseline. Also printing all the results to the baseline helps in avoiding the need for counting the passing checks in the test file.